src/tests/suite/daos_degrade_ec.c

```diff
@@ -302,7 +302,11 @@ degrade_small_sub_setup(void **state)
 
 	rc = test_setup(state, SETUP_CONT_CONNECT, true,
 			DEGRADE_SMALL_POOL_SIZE, 6, NULL);
-	return rc;
+	if (rc)
```

Comment:
well, we do not want to fail the test due to pool creation failure(most likely due to the environment issue, for example does not have enough ranks/targets etc), since the test is not about pool creation. so it does not return failure here, instead it is trying to skip.

But there is no easy way to skip in setup, so I decided to return zero here, but check the pool status again in real tests and skip there.

Hmm, if there is system query, which we can use in our test code,  then it will be much easier here. but...